test-types/jasmine/types-jasmine.test.ts

```diff
@@ -0,0 +1,848 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+// Desired since we do not want to overwrite the global `expect` from Jasmine
+import { expect as wdioExpect } from 'expect-webdriverio'
+describe('type assertions', () => {
```

Comment:
Maybe we should transform this into a [type test](https://vitest.dev/guide/testing-types.html)?

Reply:
Don't worry, we can go as is, not a blocker to move this forward.